dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/JarLinkageReport.java

```diff
@@ -46,4 +46,26 @@ static Builder builder() {
     abstract Builder setMissingFieldErrors(Iterable<LinkageErrorMissingField> value);
     abstract JarLinkageReport build();
   }
+  
+  @Override
+  public String toString() {
+    StringBuilder builder = new StringBuilder();
+    int totalErrors = getMissingClassErrors().size() + getMissingMethodErrors().size()
+        + getMissingFieldErrors().size();
+    builder.append(getJarPath().getFileName() + " (" + totalErrors + " errors):\n");
```

Comment:
Might be nice to use a `Formatter` instead of a `StringBuilder`. Up to you of course.